src/debuginfo.cpp

```diff
@@ -779,6 +798,7 @@ void jl_getFunctionInfo(const char **name, size_t *line, const char **filename,
 #endif
 #endif
         lookup_pointer(context, name, line, filename, pointer, 1, fromC);
+        delete context;
```

Comment:
Does this need an `ifdef !_OS_WINDOWS_` around it? See `context = it->second.ctx;` above.